assets/js/components/GoogleChart.js

```diff
@@ -225,24 +225,44 @@ export default function GoogleChart( props ) {
 		}
 	}
 
+	// Note the `typeof WebFont === 'undefined'` check for every `fontName`
+	// property. If the Google `WebFont` global exists and we try to use
+	// the `fontName` property in a Google Chart, we'll encounter an
+	// exception and the entire dashboard will crash.
+	//
+	// Our "workaround" is not to load custom fonts when the `WebFont`
+	// global exists, as these are usually very small UI elements
+	// and there's no clear way to get them to load without errors
+	// when the `WebFont` global/script is loaded.
+	//
+	// See: https://github.com/google/site-kit-wp/issues/5572
 	merge( chartOptions, {
 		hAxis: {
 			textStyle: {
-				fontName: 'Google Sans Text',
+				fontName:
+					typeof WebFont === 'undefined'
+						? '"Google Sans Text", "Helvetica Neue", Helvetica, Arial, sans-serif'
+						: undefined,
```

Comment:
How about we extract this to a helper function? This would be  a bit cleaner in each instance and then we also don't need to duplicate the reasoning behind it in each use. Most people will copy whatever we already use somewhere else anyways so it should be safe for future use as well.